src/utils/theme-settings.js

```diff
@@ -27,7 +28,7 @@ export const { baseUrl } = window;
  * @return {Promise<{}>}
  */
 export async function fetchThemeSettings() {
-  const url = new URL(`${baseUrl}theme-settings.json`, window.location.href);
+  const url = resolveAbsoluteUrl('/theme-settings.json');
```

Comment:
Here you are also removing the baseUrl, same question as above

Reply:
https://github.com/apple/swift-docc-render/pull/357#discussion_r901793839